pom.xml

```diff
@@ -36,13 +36,12 @@
 		<dependency>
 			<groupId>commons-io</groupId>
 			<artifactId>commons-io</artifactId>
-			<scope>compile</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.apache.httpcomponents</groupId>
 			<artifactId>httpclient-osgi</artifactId>
 			<version>4.3.3</version>
```

Comment:
It was a revert to original state. Seems I accidentally declared it as compiled in https://github.com/comsode-uv-plugins/e-skGovCRP/commit/fd8e7c047219249d99c1d8136e5b4801527d53e1.